### ISSUE TITLE
Support Custom command module

### ISSUE THREAD

**Issue by FlyinPancake** [LivacoNew/CrabFetch#30]
**What would you like to see added to CrabFetch?**
I'd love to have some custom modules

**What use would this be?**
For niche use-cases like rpm-ostree deployments that the wider linux world does not care about, crabfetch could support external modules.

**Additional Comments**
The module I'd like to convert from fastfetch
```jsonc
{
            "type": "command",
            "key": " 󱋩",
            "text": "/usr/libexec/bazzite-fetch-image"
},
```

**Comment by LivacoNew** [comment]
This already links into another plan I already have which is per-module configuration, e.g you could have two `displays` modules but one only outputs the res, while the other grabs the DRM name (bad example but i suck at examples so deal with it)

TOML allows [mixed type arrays](https://toml.io/en/v1.0.0#array) for this to be embedded directly into the `modules` array, but I'm not sure how the config crate I use would let me use it. 

Either way, once that's implemented I can start looking into this.

**Comment by FlyinPancake** [comment]
So, if I read this correctly, you want to embed all module settings in the modules array instead of the placeholder strings?

**Comment by LivacoNew** [comment]
Not quite, moreso you can override the module's settings for a single entry. Example;

```
modules = [
    "displays",     # Uses the default displays
    {"displays", {
        title = "But this uses a different title, but is the same module"
    }}
]
```

This way your custom commands can simply be done like so;
```
modules = [
    {"custom", {
        title = "Custommm",
        exec = "/some/where/myscript.sh"
    }}
]
```

**Comment by jeevithakannan2** [comment]
This doesn't work now

**Comment by LivacoNew** [comment]
I've finally implemented this, as a new inline module, usable like so:

![Image](https://github.com/user-attachments/assets/4b2f9dde-db3c-49ec-92ea-18e6b78b6d55)

![Image](https://github.com/user-attachments/assets/08d9e821-6390-477a-8929-52bf3efb487d)

Sorry it look so long. It does have limitations but it should work for the 99% of people's "invoke script at x location" use-case.
